src/create_sphere_and_arrow.cpp

```diff
@@ -0,0 +1,135 @@
+#include<ros/ros.h>
+#include "reuleaux/WorkSpace.h"
+#include<octomap/octomap.h>
+#include<octomap/MapCollection.h>
+#include <octomap/math/Utils.h>
+#include <reuleaux/sphere_discretization.h>
+#include <tf2/LinearMath/Quaternion.h>
+#include "geometry_msgs/PoseArray.h"
+#include<map>
+#include<reuleaux/kinematics.h>
+
+using namespace octomap;
+using namespace std;
+using namespace octomath;
+using namespace sphere_discretization;
+using namespace kinematics;
+
+int main(int argc, char **argv)
+{
+    ros::init(argc, argv, "workspace");
+    ros::NodeHandle n;
+  
+    
+
+    ros::Publisher workspace_pub = n.advertise<reuleaux::WorkSpace>("workspace", 1000);
+  //ros::Rate loop_rate(10);
+  ros::Rate poll_rate(100);
+while(workspace_pub.getNumSubscribers() == 0)
```

Comment:
The formatting on this file makes it hard to follow.  Make sure to follow the C++ style guide found [here](http://wiki.ros.org/Industrial/DevProcess).  You can set your editor preferences or post format the code.